### DISCUSSION TITLE
Display VPC with "Time after last dose"

### DISCUSSION THREAD

**Discussion by akosuaagyeman** [nlmixr2/nlmixr2#48] [Q&A]
Hi nlmixr Team,

Please how do I display VPC with **Time after last dose** instead of **Time after first dose** on the x-axis for nlmixr2?

Many thanks

**Comment by mattfidler** [comment]
While there is a way, it is easier to implement a feature than to explain it.

**Reply by akosuaagyeman** [reply]
Thanks Matt for the response.
I am not quite clear with what you mean by "it is easier to implement a feature than to explain it".
Can you please explain further?
Many thanks.

**Reply by mattfidler** [reply]
As of now, you can use `vpcPlotTad()`, `vpcPlotTad(pred_corr=TRUE)` and `vpcCensTad()`  for the 3 different types of vpc with some nlmixr support built-in.

Now it will overlay the `lloq` value in the plots when you have censoring.

**Reply by mattfidler** [reply]
> Thanks Matt for the response. I am not quite clear with what you mean by "it is easier to implement a feature than to explain it". Can you please explain further? Many thanks.

Mostly it required the `tad` to be in the simulation as well as in the observations.  While you can do it (and provide an example), by the time it is coded it should really be added to the `nlmixr2plot`

**Reply by akosuaagyeman** [reply]
Many thanks Matt. I'm now able to make the plots with the vpcPlotTad().
One thing I realised is that, vpcPlotTad() did not run plus I get error message when I didn't have a column labelled 'tad' in my datafile prior to model fitting. Many thanks again for the help.
